sagan/models/sagan.py

```diff
@@ -0,0 +1,104 @@
+from absl import app
+from absl import flags
+
+from ml_collections.config_flags import config_flags
+
+import tensorflow as tf
+from tensorflow import keras
+
+from models.generator import build_generator
+from models.discriminator import build_discriminator
+from sagan.utils import w_loss
+
+FLAGS = flags.FLAGS
+config_flags.DEFINE_config_file("experiment_configs")
+
+class SelfAttentionGAN(keras.Model):
+    def __init__(self, FLAGS.experiment_configs.latent_dim, FLAGS.experiment_configs.gp_weight, **kwargs):
```

Comment:
`FLAGS.experiment_configs.latent_dim` and `FLAGS.experiment_configs.gp_weight` aren't valid literal names in python. It should be 

```python3
def __init__(self, configs: ml_collections.ConfigDicts, **kwargs):
    self.latent_dim = configs.latent_dim
    self.gp_weight = configs.gp_weight
    self.generator = build_generator(configs.image_size, configs.latent_dim)
    self.discriminator = build_discriminator(configs.image_size)
```